packages/core/src/components/text-input/text-input.css

```diff
@@ -258,18 +258,23 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  margin-right: var(--spacing-s);
+  margin-right: calc(var(--spacing-s) - var(--spacing-xs) / 2);
   font-size: 1rem;
 }
 
 .hds-text-input__button {
-  padding: 0;
   appearance: none;
-  background: transparent;
+  background: none;
+  border: none;
+  color: inherit;
   cursor: pointer;
-  border: 0;
-  width: 1.5rem;
-  height: 1.5rem;
-  overflow: hidden;
-  font-size: 1rem;
+  font: inherit;
+  color: var(--dropdown-color-default);
```

Comment:
Color is defined twice? And this color definition might be the reason for loki images change because as Hannu pointed out, the color is different in new images. It used to be #0000 (color-black) but it has changed to #1a1a1a (black-90). Because it's a lighter color it might seem like there is an anti-aliasing effect because the difference is more noticeable to the naked eye around the borders of the icon.

Reply:
Hmm, I removed the duplicate color definition but that had no difference in loki tests.

Reply:
Oh, there was no inheritance before. I added it to unify the input stylings. Should we leave it as it is now or revert the `color: inherit`?

Reply:
Reverted. Changed files of this PR went from 24 to 4 😆